docs/src/main/asciidoc/kafka-schema-registry-avro.adoc

```diff
@@ -71,6 +71,10 @@ In this guide, we will use the ones provided by Apicurio.
     <artifactId>apicurio-registry-serdes-avro-serde</artifactId>
     <version>2.0.0.Final</version>
 </dependency>
+<dependency>
```

Comment:
```suggestion
<!-- Apicurio Registry libraries use Keycloak client, which requires Apache HTTP Client -->
<dependency>
```

That's probably the only thing I'd do. For Confluent, we also only document what is needed (few lines below) and don't try to be smart. Documenting it should be enough for Apicurio too.